boilerplate/flyte/end2end/end2end.sh

```diff
@@ -9,11 +9,4 @@ set -e
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
-OUT="${DIR}/tmp"
-rm -rf ${OUT}
-git clone https://github.com/flyteorg/flyte.git "${OUT}"
-
-pushd ${OUT}
-
-make end2end_execute
-popd
+python ./boilerplate/flyte/end2end/run-tests.py $VERSION $PRIORITIES
```

Comment:
This command should look similar to https://github.com/flyteorg/flytetools/blob/master/flytetester/functional-tests/run.sh#L10. Notice the presence of:
1. the `FLYTE_SDK_USE_STRUCTURED_DATASET=TRUE` prepended to the command.
2. the config file
3. the string "core" to indicate that the script will fail if any of the core tests fail.